src/main/java/net/consensys/orion/config/Config.java

```diff
@@ -659,6 +683,73 @@ private static Schema configSchema() {
         "TLS known servers for the client. This contains the fingerprints of public keys of other nodes that this node has encountered for the ca-or-tofu, tofu and whitelist trust modes.",
         null);
 
+    schemaBuilder.addString(
+        "clientconnectiontls",
+        "off",
+        "TLS status. Options:\n"
+            + "\n"
+            + "   - strict: All connections to the client connection of this node must use TLS with client authentication\n"
+            + "       See the documentation 'clientconnectiontlsservertrust'\n"
+            + "   - off: Mutually authenticated TLS is not used for in- and outbound\n"
```

Comment:
given this is what people are actually reading, i'd suggest changing 'in-' to 'inbound'

Reply:
wanted to not change as much as possible - so have literally cut-and-paste existing descriptions.

Reply:
fair enough, i thought this was 'new'.. all good